.github/workflows/ci.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           pip install -r requirements-dev.txt
       - name: Check formatting
         run: black --check gnomad tests
+      - name: Check imports
+        run: isort --check-only gnomad tests
```

Comment:
basic question, what is the difference between `--check` (used above for black) and `--check-only`?

Reply:
it's the same thing, they check but don't reformat, just named differently for black and isort